teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,58 +4,83 @@
    "cell_type": "markdown",
```

Comment:
<p>[Again I didn't run it locally yet, so I don't see the actual figure]</p><p> </p><p>Anyhow could you add a few sentences on what we see (probably since it is a kinase, the ligand shows 1 (or 2 or 3) Hbonds to the hinge region, and maybe some hydrophobic backpocket interactions or so.</p><p> </p><p>And one could link to KLIFS <a href="https://klifs.net/details.php?structure_id=7308" target="_blank">3poz</a> and compare the main findings to the KLIFS interaction fingerprint.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>